cobra/__init__.py

```diff
@@ -972,7 +973,7 @@ def __init__(self, URI, retrymax=None, timeout=None, **kwargs):
             self._cobra_sockpool = queue.Queue()
             # timeout reqeuired for pool usage
             if not self._cobra_timeout:
-                self._cobra_timeout = 60
+                self._cobra_timeout = 120
```

Comment:
this timeout i'm not absolutely sure is necessary, or the potential ramifications.  some large memory-maps have taken >30 seconds to send by themselves, and before tweaking this and the other timeouts, i was getting lost messages.

Reply:
not an easy way to do it currently.  it seems to want to be in config data, but at this level, cobra doesn't have easy access to the config.

perhaps we should consider making one config global for the entire runtime.  while you could have multiple workspaces instantiated at once, there are some things that might appreciate the default config?

let's talk about how to do this well.